doc/source/whatsnew/v0.23.1.txt

```diff
@@ -45,6 +45,7 @@ Bug Fixes
 ~~~~~~~~~
 
 - tab completion on :class:`Index` in IPython no longer outputs deprecation warnings (:issue:`21125`)
+- fix normalize to not except when json_normalize is fed a key with null value on inner most key (:issue:`21158`)
```

Comment:
move to IO section, use a ref to ``json_normalize``.  see if you can describe this is more plain english.